ComponentKit/Core/Scope/CKComponentScopeHandle.mm

```diff
@@ -184,6 +185,15 @@ - (BOOL)acquireFromComponent:(id<CKComponentProtocol>)component
   }
 }
 
+- (void)aquireFromComponentAssertIfWrong:(id<CKComponentProtocol>)component
```

Comment:
Why is this different from above?

Reply:
As we create the scope explicitly for a specific component, we don't need to check `isMemberOfClass` and `_acquired` - which could make it faster.